Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-pan-os-python = "*"
+pan-os-python = ">=1.1.0"
```

Comment:
pipenv is set to pull the latest version of this package, so keep this set to `"*"`.